docs/run-core-node/installation.mdx

```diff
@@ -3,7 +3,7 @@ title: Installing
 sidebar_position: 30
 ---
 
-There are three ways to install Stellar Core: you can use a Docker image, use pre-built packages, or build from source. Using a Docker image is the quickest and easiest method, so it's a good choice for a lot of developers, but if you're running Stellar Core in production you may need to use packages or, if you want to sacrifice convenience for maximum control, build from source. Whichever method you use, you should make sure to install the latest [release](https://github.com/stellar/stellar-core/releases) since releases are cumulative and backwards compatible.
+There are multiple ways to install Stellar Core: you can use a Docker image, pre-built packages, helm-chart, or build from source. Using the Docker image is the quickest and easiest method, so it's a good choice for a lot of developers, but if you're running Stellar Core in production you should consider the helm chart or packages, if you want to sacrifice convenience for maximum control, build from source. Whichever method you use, you should make sure to install the latest [release](https://github.com/stellar/stellar-core/releases) since releases are cumulative and backwards compatible.
```

Comment:
This paragraph list few ways to install Stellar Core. The "grocery list" here would ideally match, in both name & order the subsequent topic titles. Could you align these ?

Reply:
re-aligned the text here for drill-down into matching sections - https://github.com/stellar/stellar-docs/pull/216/commits/4cbc536dc93f26d00ee1a9871d9a19e623a11f24